README.md

```diff
@@ -167,60 +173,94 @@ with open(path/to/model_file.pickle.dat, 'rb') as model_file:
 ```
 
 ## FAQ:
-- Will CAPICE support CADD 1.6 and Genome Build 38?
+__Question:__  
+Will CAPICE support CADD 1.6 and Genome Build 38?
+
+__Answer:__  
+No (CADD 1.6) and yes (Genome Build 38). CADD has moved on to Snakemake and is quite slow.
+It also limits us on updating VEP for improved and bugfixes on features. However, CAPICE will support genome build 38.
 
-No. CADD has moved on to Snakemake and is quite slow. It also limits us on updating VEP for improved and bugfixes on
-features. However, CAPICE will support genome build 38.
+---
 
-- These scores are nice and all, but what do they really mean for this particular variant?
+__Question:__  
+These scores are nice and all, but what do they really mean for this particular variant?
 
+__Answer:__  
 CAPICE bases it's scoring on the training that it was provided with. A score is assigned based on features the model
 learned to recognize during training. There are plans to make a "Capice Explain Tool" which will tell how a score came
 to be.
 
-- Training a new model failed with an error in `joblib.externals.loky.process_executor.RemoteTraceback`
+---
+
+__Question:__   
+Training a new model failed with an error in `joblib.externals.loky.process_executor.RemoteTraceback`
   with `ValueError: unknown format is not supported`. Why?
 
+__Answer:__  
 This could possibly originate from a low sample size during the cross validation in RandomSearchCV.
 Please [contact us](https://github.com/molgenis/capice/issues) for further help.
 
-- I'm on Windows and installing XGBoost fails with the PIP
+---
+
+__Question:__  
+I'm on Windows and installing XGBoost fails with the PIP
   error `“No files/directories in C:\path\to\xgboost\pip-egg-info (from PKG-INFO)”`. Am I doing something wrong?
 
-Unfortunatly, XGBoost does not cooperate well with Windows. You might want to try to
+__Answer:__  
+Unfortunately, XGBoost does not cooperate well with Windows. You might want to try to
 install [Setuptools](https://pypi.org/project/setuptools/) before you attempt to install the dependencies. If that does
 not work either, we suggest you use either a Unix style virtual machine or, if you are using Windows 10,
 the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10) is also available in the
 Windows Store for free, which is guaranteed to work.
 
 You could also use CAPICE through the [Singularity Container](https://download.molgeniscloud.org/downloads/vip/images/).
 
-- I'm getting
+---
+
+__Question:__  
```

Comment:
I'm in doubt whenever this Question is still relevant. Since CAPICE 3.0.0, we're using F-strings as well, which are not supported in Python3.5 or earlier versions. Furthermore, we've changed the way that the model is called to perform it's prediction, which is not compatible with the model produced by Li et al. To my knowledge, the models for 3.0.0 will not generate this issue if installed properly. Maybe scrap the question?

Reply:
Sure, we can always re-add it in case issues are created regarding this error. Besides, `setup.py` installs a very specific xgboost version, so an incorrect xgboost version error should only occur if a user installed CAPICE through non-conventional methods.